static/messages.less

```diff
@@ -18,6 +18,7 @@ ul.background-message {
   padding: 0;
 
   li {
+    line-height: 42px;
```

Comment:
We should use some manner of relative size here. Pretty sure this is well-behaved for non-default font sizes. /cc @simurai. 

---
_The choice of 42 is... "cute"_ 😁